shuup/admin/templates/shuup/admin/base.jinja

```diff
@@ -79,20 +79,36 @@
         {% endblock %}
         {% block post_content %}{% endblock %}
         {% endif %}
-        <script src="{{ shuup_static("shuup_admin/js/vendor.js") }}"></script>
-        <script src="{{ shuup_static("shuup_admin/js/base.js") }}"></script>
-        {% block extra_js %}{% endblock %}
-        {% if messages %}
+
+        <script src="{{ shuup_static('shuup_admin/js/vendor.js') }}"></script>
+        <script src="{{ shuup_static('shuup_admin/js/base.js') }}"></script>
+
+        {% block init_scripts %}
         <script>
-            (function(){
-                if(!window.Messages) return;
-                {% for message in messages -%}Messages.enqueue({
-                    tags: {{ message.tags|json }},
-                    text: {{ (message.tags.capitalize() ~ "! " ~ message.message)|json }}
-                });{% endfor %}
-            }());
+            if (window.ShuupAdminEditors) {
```

Comment:
Can't this be inside base.js? Since it does not require anything from the jinja template?